tests/test_hash_table.py

```diff
@@ -0,0 +1,108 @@
+# coding=utf-8
+import pytest
+
+
+@pytest.fixture(scope='session')
+def words():
+    d = {}
+    with open('test_data/words', 'r') as f:
+        for word in f:
+            word = word.strip()
+            if word not in d:
+                d[word] = len(d)
+    return d
+
+
+def test_words(words):
+    from data_structures.hash_table import HashTable
+    test_hash_table = HashTable()
+    for k, v in words.items():
+        test_hash_table.set(k, v)
+    for k, v in words.items():
+        assert test_hash_table.get(k) == v
+
+
+TEST_SIZE = [
+    # ["", 1024],
+    [0, 0],
+    [1, 1],
+    [1024, 1024],
+    [2048, 2048],
+]
+
+
+@pytest.mark.parametrize("size, result", TEST_SIZE)
+def test_hashtable_init(size, result):
+    from data_structures.hash_table import HashTable
+    test_hash_table = HashTable(size)
+    assert len(test_hash_table._storage) == result
+
+def test_hashtable_init_empty():
+    from data_structures.hash_table import HashTable
+    test_hash_table = HashTable(hash_size=0)
+    assert len(test_hash_table._storage) == 0
+
```

Comment:
Test that if you try to use a key that isn't currently in your hash table, you return None